docs/2.7.0/docs/canton/usermanual/security.rst

```diff
@@ -25,35 +25,30 @@ Supported Cryptographic Schemes in Canton
 Within Canton we use the cryptographic primitives of signing, symmetric and
 asymmetric encryption, and MAC with the following supported schemes:
 
-Signing:
-
-- Ed25519 (default)
-- ECDSA with P-256 and P-384
-- SM2 (experimental)
-
-Symmetric Encryption:
-
-- AES128-GCM (default)
-
-Asymmetric Encryption:
-
-- ECIES on P-256 with HMAC-SHA256 and AES128-GCM (default)
-- ECIES on P-256 with HMAC-SHA256 and AES128-CBC
-- RSA 2048 with OAEP using SHA-256
-
-MAC:
-
-- HMAC with SHA-256
+.. image:: ./images/canton_supported_schemes.png
```

Comment:
please use sphinx tables and not a picture (easier to change, searchable): https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#grid-tables